### PR TITLE
Issues 159 & 164: UUID as ID and changed alert message for container that is back and running

### PR DESCRIPTION
**Pull request by jakubsolecki**
https://github.com/agh-docker-monitoring/dockiera-app/issues/159 and https://github.com/agh-docker-monitoring/dockiera-app/issues/164

@skupien @kraleppa UUID breaks forntend, fixes are needed

### CODE REVIEW

**Comment by kraleppa** [comment]
Probably because of typescript - should be easy fix (I hope...)

**Comment by kraleppa** [comment]
@Ilargi12 some fixes on agent side might be needed too

File: frontend/src/redux/hostsSlice.ts [hunk 1]
```diff
@@ -3,10 +3,11 @@ import { RootState } from "./store";
 import { fetchHosts } from "../common/api";
 import { Host, HostSummary } from "../common/types";
 
+//We need @Patryk Skupień here
```
Comment (kraleppa) [inline frontend/src/redux/hostsSlice.ts:6]: @skupien could you please check this file... below you are doing some weird stuff with ids assuming that they're integer values and it won't work after change to uuid

**Review by kraleppa — CHANGES_REQUESTED** [review]
We need @skupien here

**Comment by skupien** [comment]
Fixed by replacing [] with {}

**Review by skupien — APPROVED** [review]

**Pull request merged**